app/Models/Training/WaitingList/WaitingListAccount.php

```diff
@@ -176,6 +178,21 @@ public function getEligibilityAttribute()
         return $this->atcHourCheck() && $this->allFlagsChecker() && $this->current_status->name == 'Active';
     }
 
+    public function getTheoryExamPassedAttribute(): bool
+    {
+        if ($this->waitingList->department === WaitingList::PILOT_DEPARTMENT || ! $this->waitingList->cts_theory_exam_level) {
+            return false;
```

Comment:
I feel like this function should return `bool?` - feels like we should return `null` if no theory exam level set, and display this on the UI (maybe like a - )? Returning false in this case feels like we could give the user the wrong impression.

Equally, why are we specifically excluding pilot lists from being able to do this?

Reply:
ATC are the only ones using theory exams in CTS at the moment.

I've got it so that only ATC lists have the column included on the front-end, which is why I thought a true/false was going to be sufficient for now.

Reply:
I'm indifferent here but my initial read (before seeing Alex's comment) was that null would better represent an absence of any requirement. Passing false, you could argue, is wrong and we should force it to true to pass.

Reply:
Have made some adjustments to return null in certain conditions:

https://github.com/VATSIM-UK/core/pull/2824/commits/b08f888d7de73b38df763253a77378706fb73835